example/pico-ssram/firmware.c

```diff
@@ -12,10 +12,17 @@
 #define DATA_LEN 8
 #define START_ADDR 0
 
+static volatile int g_sequence_count;
+
+void async_callback(void) {
+    ++g_sequence_count;
+}
+
 int main() {
     ice_sdk_init();
     stdio_init_all();
-    ice_ssram_init(DMA_IRQ_1 /* Pass -1 for synchronous mode */);
+    ice_ssram_init(10*1000*1000, DMA_IRQ_1 /* Pass -1 for synchronous mode */);
+    ice_ssram_set_async_callback(async_callback);
```

Comment:
What about having the async_callback passed as an argument to `ice_ssram_init`?
Was there some reason to change the async_callback that I missed?

Reply:
My idea is you might want to have different callbacks for e.g. reads vs writes. Actually, I considered making the async_callback be an additional parameter to ice_ssram_read_async() and ice_ssram_write_async(). That would help prevent bugs relating to forgetting to set the async_callback to null and then having it unexpectedly called later.